### PR TITLE
TKO-31 BE: Set up student profiles & storing profile form info

### PR DESCRIPTION
**Pull request by nixy97**
# Description
Create a Student Information migration and store form data into the student_info database
  
# Testing
Once on this branch, go to this url: http://localhost:8080/profile/create fill in the form and submit. Then check the student_info table to see if your information was stored

# Installation
Run composer install

### CODE REVIEW

File: app/Http/Controllers/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            'units'=> 'required|integer',
+            'grad_date' => 'required',
+            'college'=>'required',
+            'bio'=> 'required',
+            'research' => 'required',
+            'fun_fact'=>'required',
+            'academic_interests' => 'required'
+        ]);
+
+        //StudentInfo::create($request->all());
```
Comment (MikkalMc) [inline app/Http/Controllers/ProfileController.php]: remove comments

File: app/Http/Controllers/ProfileController.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers;
```
Comment (MikkalMc) [inline app/Http/Controllers/ProfileController.php:3]: Seperate controller logic into a service called ProfileService

Comment (MikkalMc) [reply app/Http/Controllers/ProfileController.php:3]: After creating service, use dependency injection to inject service into controller

File: .idea/badges.iml [hunk 3]
```diff
@@ -4,6 +4,7 @@
     <content url="file://$MODULE_DIR$">
```
Comment (MikkalMc) [inline .idea/badges.iml]: I think the folder .idea should be added to your git ignore

File: routes/web.php [hunk 4]
```diff
@@ -23,5 +23,7 @@
     return File::get(public_path() . '/docs/assets/js/*.js');
 });
 
-Route::get('/{any}', 'SpaController@index')->where('any', '.*');
+Route::resource('profile', 'ProfileController');
+
+//Route::get('/{any}', 'SpaController@index')->where('any', '.*');
```
Comment (alecnicolas) [inline routes/web.php]: Why was this commented out?

**Review by alecnicolas — DISMISSED** [review]

File: routes/web.php (hunk 4 above)
Comment (nixy97) [reply routes/web.php]: Oh it wasn't supposed to be. I'll fix it

**Review by MikkalMc — APPROVED** [review]

**Review by alecnicolas — APPROVED** [review]
🤓

**Pull request merged**